modules/ROOT/pages/overview-apikit-for-rest.adoc

```diff
@@ -3,4 +3,25 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-TBD
\ No newline at end of file
+APIkit for REST is a powerful framework designed to simplify the development and integration of RESTful APIs. REST (Representational State Transfer) is an architectural style for building web services, and APIkit for REST provides a comprehensive set of tools and features to streamline REST API development while ensuring compliance with industry standards.
+
+APIkit for REST fully supports RAML (RESTful API Modeling Language) and OAS (OpenAPI Specification), which are widely adopted formats for describing RESTful APIs. By leveraging RAML or OAS, you can define your API contracts, enabling consistent design, enhanced collaboration, and clear documentation.
+
+Features:
+
+* *Automatic Validation*: APIkit for REST automatically validates incoming requests and outgoing responses against your API contracts defined in RAML or OAS. This ensures that your APIs adhere to the specified rules and constraints, enhancing the overall quality and reliability of your API implementation.
```

Comment:
APIkit does not validate outgoing responses for any protocol.
```suggestion
* *Automatic Validation*: APIkit for REST automatically validates incoming requests against your API contracts defined in RAML or OAS. This ensures that requests adhere to the specified rules and constraints of the API, enhancing the overall quality and reliability of your API implementation.
```

Reply:
Thanks!